modules/wmco-prerequisites.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * windows_containers/understanding-windows-container-workloads.adoc
+
+[id="wmco-prerequisites_{context}"]
+= Windows Machine Config Operator (WMCO) prerequisites
```

Comment:
ISG isn't prescriptive about this, but discourages including an abbreviation in headings.  For me, it's in "Abbreviations" > "Headings and titles" around p46-7.  Up to you, but I'd prolly remove "WMCO".